node/cmd/guardiand/adminnodes.go

```diff
@@ -115,6 +115,7 @@ func runListNodes(cmd *cobra.Command, args []string) {
 		{"Sei", vaa.ChainIDSei},
 		{"Scroll", vaa.ChainIDScroll},
 		{"Mantle", vaa.ChainIDMantle},
+		{"MonadDevnet", vaa.ChainIDMonadDevnet},
```

Comment:
Nit (and you don't have to do it), but can you move this down to the other testnet only chains (all the sepolia ones)